### PR TITLE
Adding Github Issues templates

### PR DESCRIPTION
**Pull request by alex-rakowski**

### CODE REVIEW

**Comment by alex-rakowski** [comment]
The commit history look a little strange, I think I forgot update my dev branch outside of this commit, but they're just syncing to current py4DSTEM dev branch

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+**To Reproduce**
+Steps to reproduce the behavior, please be as general as possible, and ideally recreate a minimal reproducible example:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+
+**py4DSTEM version** 
+It can be accessed by running `py4DSTEM__version__`
```
Comment (sezelt) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: missing a `.`

**Review by sezelt — COMMENTED** [review]
I am a big fan of this. There is just one missing `.` (the line `py4DSTEM__version__`) that needs to be fixed

**Comment by alex-rakowski** [comment]
Added the dot. Thanks for catching it Steve

**Comment by bsavitzky** [comment]
Awesome, thanks Alex!  The new files with the templates look good, however the docs build is check is failing....it looks like it's this issue with that walrus operator in wpf.py line 188 again :|  I think either replacing the walrus, or modifying the readthedocs build so that it uses only Python 3.8+ environments, should fix the issue

**Comment by bsavitzky** [comment]
Amazing ty Alex :)

**Review by bsavitzky — APPROVED** [review]

**Pull request merged**